src/cow-react/modules/application/dumb/Widget/index.tsx

```diff
@@ -1,23 +1,32 @@
 import styled from 'styled-components/macro'
 import { BodyWrapper as BodyWrapperMod } from '@src/pages/AppBody'
-import { transparentize } from 'polished'
 
-export const BodyWrapper = styled(BodyWrapperMod)`
+const Wrapper = styled(BodyWrapperMod)`
```

Comment:
I thought you said no styles in dumb components?
Refactor still to come or I got the rule wrong?